spec/03-plugins/23-rate-limiting/05-integration_spec.lua

```diff
@@ -88,209 +88,152 @@ describe("Plugin: rate-limiting (integration)", function()
     },
   }
 
+  -- it's set smaller than SLEEP_TIME in purpose
```

Comment:
should **also** we test with a SYNC_RATE that is higher than SLEEP_TIME?

Reply:
`SLEEP_TIME` is a variable serving for test purposes, making the test wait for the counters to sync

Reply:
I suppose you confused it with the sleep time we used to wait for sync lock...

Reply:
whoops, sorry, got it.